index.html

```diff
@@ -4,15 +4,42 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+<!--End-->
     <link rel="stylesheet" href="style.css">
     <title>Setup and Mobile first</title>
 </head>
 <body>
-    <div class="container">
-        <div class="box">B 1</div>
-        <div class="box one">B 2</div>
-        <div class="box">B 3</div>
-        <div class="box">B 4</div>
-    </div>
+    <nav class="Navbar">
+        <div class="top-logo">
+            <h2>My Portfolio</h2>
+        </div>
+        <div class="menu-icon">
+            <div class="menu-vector"></div>
+            <div class="menu-vector"></div>
+            <div class="menu-vector"></div>
+        </div>
+    </nav>
+    <main>
+        <div class="informative-top">
+            <div class="top-heading">
+                <h1>Hello, My name is Abbas Glad to see you!</h1>
+            </div>
+            <div class="heading-description">
+                <p>I'm a web developer! I can help you to create a Product or Portfolio website for you. Check out my website and contact me if you find it great.</p> 
+            </div>
+            <div class="touch-links">
+                <h3><a href="#">LET'S CONNECT</a></h3>
+                <div class="social-bar">
+                    <a href="https://twitter.com/Abbas_sDev" class="connect"><img src="image-resources/twitter.png" alt="Twitter logo"></a>
+                    <a href="https://www.linkedin.com/in/abbas-sarwar-4a0b16257/" class="connect"><img src="image-resources/linkedIn.png" alt="LinkedIn logo"></a>
+                    <a href="https://github.com/AbbasSarwar" class="connect"><img src="image-resources/Github.png" alt="Github logo"></a>
```

Comment:
- [ ] Please make sure you add all the social media icons as mentioned in the [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md)

> Your website should look exactly like the design (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). 

**_Your design_**

<img width="120" alt="Screenshot 2023-02-16 at 11 21 54 AM" src="https://user-images.githubusercontent.com/44798044/219299996-034a9f8b-e034-4299-9e33-301a5d1c3675.png">

**_Figma design_** 

<img width="175" alt="Screenshot 2023-02-16 at 12 36 03 PM" src="https://user-images.githubusercontent.com/44798044/219300044-1fc2c0d4-7ba8-4454-83bd-7ab6d26caec7.png">